gemfiles/rails7.0.gemfile

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Should this be added to the test matrix?

Reply:
I... I thought I did?!?

Reply:
OK. Added and failing but green for CI